packages/vector_graphics_compiler/lib/vector_graphics_compiler.dart

```diff
@@ -116,12 +116,20 @@ Future<Uint8List> encodeSVG(String input, String filename) async {
         break;
       case DrawCommandType.vertices:
         final IndexedVertices vertices =
-            instructions.vertices[command.objectId];
+            instructions.vertices[command.objectId as int];
```

Comment:
drop cast, add assert that objectId >= 0 and < vertices.length?

Same for path